### PR TITLE
Fixing bugs  and adding new translations

### PR DESCRIPTION
**Pull request by santiagobiali**

### CODE REVIEW

File: locale/validation.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ validation:
   json: "O :attribute deve ser uma string JSON válida."
   max:
     numeric: "O :attribute não pode ser maior do que :max."
-    file: "O :attribute não pode ser maior do que :max kilobytes."
+    file: "O :attribute não pode ser mribute não pode ser maior do queaior do que :max kilobytes."
```
Comment (kassius) [inline locale/validation.yml]: typo

File: locale/flarum-likes.yml [hunk 2]
```diff
@@ -16,14 +16,14 @@ flarum-likes:
...
+      post_liked_text: "{username} curtiu seu post|{username} curtiram seu post"  # Can be pluralized to agree with the number of users!
       others_text: => core.ref.some_others
 
     # These strings are displayed beneath individual posts.
     post:
-      like_link: Like
+      like_link: Curtir
       liked_by_text: "{users} curtiu isso.|{users} curtiram isso."
-      liked_by_self_text: "{users} like this."  # Can be pluralized to agree with the number of users!
+      liked_by_self_text: "{users} curtiu isso.|curtiram isso"  # Can be pluralized to agree with the number of users!
```
Comment (kassius) [inline locale/flarum-likes.yml]: {users}curtiu isso|{users} curtiram isso

File: locale/flarum-mentions.yml [hunk 3]
```diff
@@ -13,14 +13,14 @@ flarum-mentions:
...
-      post_mentioned_text: "{username} respondeu ao seu post"  # Can be pluralized to agree with the number of users!
+      post_mentioned_text: "{username} respondeu ao seu post|{username} responderam ao seu post"  # Can be pluralized to agree with the number of users!
       others_text: => core.ref.some_others
       user_mentioned_text: "{username} mencionou você"
 
     # These strings are displayed beneath individual posts.
     post:
-      mentioned_by_text: "{users} respondeu à isto."       # Can be pluralized to agree with the number of users!
-      mentioned_by_self_text: "{users} responderam à isto."  # Can be pluralized to agree with the number of users!
+      mentioned_by_text: "{users} respondeu à isto.|responderam à isto"       # Can be pluralized to agree with the number of users!
```
Comment (kassius) [inline locale/flarum-mentions.yml]: {users} responderam ?

**Comment by santiagobiali** [comment]
sorry for the mistakes.

**Pull request merged**